CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Contributing
+
+## Request for changes/ Pull Requests
+You first need to create a fork of this repository to commit your changes to it. Methods to fork a repository can be found in the [GitHub Documentation](https://docs.github.com/en/get-started/quickstart/fork-a-repo).
+
+## Choose a base branch
+Before starting development, you need to know which branch to base your modifications/additions on. When in doubt, use master.
+
+| Type of change                |           | Branches              |
+| :------------------           |:---------:| ---------------------:|
+| Documentation                 |           | `master`              |
+| Bug fixes                     |           | `master`              |
+| New features                  |           | `master`              |
+| New issues models             |           | `master` |
+
+```sh
+# Switch to the desired branch
+git switch master
+
+# Pull down any upstream changes
+git pull
+
+# Create a new branch to work on
+git switch --create dev/[feature-or-fix-name]
```

Comment:
not sure about the `dev` prefix requirement. everything is a dev.
consider bugfix/ feature/ release/

Reply:
thx